lookout/core/helpers/analyzer_context_manager.py

```diff
@@ -0,0 +1,95 @@
+from argparse import Namespace
+import subprocess
+from typing import Optional, Type
+
+from lookout.core.analyzer import Analyzer
+from lookout.core.cmdline import create_model_repo_from_args
+from lookout.core.data_requests import DataService
+from lookout.core.event_listener import EventListener
+from lookout.core.helpers.server import check_port_free, find_port, LookoutSDK
+from lookout.core.manager import AnalyzerManager
+
+
+class AnalyzerContextManager:
+    """Context manager for launching analyzer."""
+
+    def __init__(self, analyzer: Type[Analyzer], db: str, fs: str,
+                 init: bool = True, data_request_address: str = "localhost:10301"):
+        """
+        Initialization.
+
+        :param db: path to sqlite database location.
+        :param fs: location where to store results of launched analyzer.
+        :param analyzer: analyzer class to use.
+        :param init: To run database initialization or not. \
+                     If you want to reuse an existing database set False.
+        :param data_request_address: DataService GRPC endpoint to use.
+        """
+        self.analyzer = analyzer
+        self.init = init
+        self._port = find_port()
+        self.data_request_address = data_request_address
+        self._sql_alchemy_model_args = Namespace(
+            db="sqlite:///%s" % db,
+            fs=fs,
+            cache_size="1G",
+            cache_ttl="6h",
+            db_kwargs={},
+        )
+        self._lookout_sdk = None
+
+    def __enter__(self) -> "AnalyzerContextManager":
+        """
+        Create context and run analyzer.
```

Comment:
```suggestion
        Create the context and run the events listener.
```

we don't run any analyzers inside!